deployment/helm/Readme.md

```diff
@@ -65,12 +65,39 @@ The Ambassador allows the routing on the TCP layer. To signal which service one
 
+| Service | Subdomain |
+|----------|----------|
+| Grafana | grafana |
+| HTTP Adapter | http-adapter |
+| Hono Device Registry | device-registry |
+| hawkBit | hawkbit |
+| Keycloak | keycloak |
+| App Store | app-store |
 
 
+# TLS 
+To encrypt the traffic from and to the Kuksa cloud one needs to make use of TLS e.g. by using HTTPS. 
 
-The decision which service is called, is signalled through the SNI field in the used certificates. 
-For the creation of these certificates the Kuksa Clouds relies on an instance of the [cert-manager](https://cert-manager.io). 
+## Cert Manager
+For the enablement of TLS one needs valid certificates for the domain under which the endpoints are hosted. In the Kuksa Cloud we use an instance of the [cert-manager](https://cert-manager.io). This tool is able to fetch new certificates and renew them by interacting with an certificate authority e.g. in the case of the standard deployment Let's encrypt.  
 
-The cert-manager requires so-called custom resource definitions (CRD) which are not installed as part of the Helm chart. To install the CRDs before installing the Helm chart one needs to run: `kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.crds.yaml`
+## enable and install Cert Manager
+The cert-manager is not enabled by default because we try to make the deployment possible even when no domain or CA is available. To include the cert-manager in the deployment go to the `Chart.yaml` file of the chart and uncomment the dependency `cert-manager`. The cert-manager further requires so-called custom resource definitions (CRD). These are installed with the cert-manager if one sets cert-manager.installCRDs to true. One can also install the CRDs by running: `kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.2/cert-manager.crds.yaml`
 
+## Cert Manager: getting certificates - ACME
+The cert-manager is able to automatically request and manage certificates from Let's encrypt through the ACME challenge. The Kuksa Cloud uses the [DNS-01 challenge](https://cert-manager.io/docs/configuration/acme/dns01/). As part of the DNS-01 challenge the cert-manager needs to prove to the CA (e.g. Let's encrypt) that it has control over the DNS entries for the domain for which it requests new certificates. The configuration and execution of the ACME challenge heavily depends on the provider of the DNS entries for the used domain. 
+
+The environment during the development of this chart was Azure. Hence, the cert-manager certificate issuer is adapted to Azure. However, we especially welcome contributions to add support for other DNS providers. 
```

Comment:
Currently, the cert-manager setup of the Eclipse Kuksa cloud only supports azure. However, ...